src/Polymorph-Widgets.package/MorphicUIManager.class/instance/lowSpaceWatcherDefaultAction..st

```diff
@@ -19,6 +19,8 @@ lowSpaceWatcherDefaultAction: theInterruptedProcess
 		newDebugSessionNamed: 'Space is low' 
 		startedAt: preemptedProcess suspendedContext.		
 	debugSession logStackToFileIfNeeded.
+	debugSession isAboutUIProcess ifTrue: [ 
+		DefaultExecutionEnvironment beActiveDuring: [self spawnNewProcess]].
```

Comment:
@estebanlm I think it should be fine. I have another problem because in #170 I added a complete commit of yours. I need to check the git history to figure out what happened